_sass/blocks/explore/_subpage.scss

```diff
@@ -0,0 +1,98 @@
+.subpage-info-wrapper {
+  @include respond-to(medium-up) {
+    margin-right: $base-padding;
+  }
+}
+
+.info-heading {
+  padding: 0 $base-padding;
+
+  @include respond-to(medium-up) {
+    padding: 0;
+  }
+}
+
+[class^=slab] {
+  padding-bottom: $base-padding;
+  padding-top: $base-padding;
+}
+
+eiti-slider {
+  margin-top: 1em;
+}
+
+.region-map {
+  display: block;
+  height: 250px;
+  margin-bottom: 1em;
+  max-height: 30vh; // 30% of the viewport height
+  width: 100%;
+}
+
+.regions .region {
+  display: none;
+
+  &.active {
+    display: block;
+  }
+}
+
+.map-legend {
+  display: flex;
+  margin-bottom: 1em;
+
+  .step {
+    border-bottom-style: solid;
+    border-bottom-width: 0.5em;
+    display: block;
+    flex: 1;
+    padding-left: 0.1em;
```

Comment:
Properties should be ordered border-bottom-style, border-bottom-width, display, flex, margin-right, padding-left